src/app/auth/auth/auth.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from "@angular/core";
+import { Router } from "@angular/router";
+import { FormControl, FormGroup, Validators } from "@angular/forms";
+import { MatDialog } from "@angular/material";
+import { AuthService } from "../../services/auth.service";
```

Comment:
![image](https://user-images.githubusercontent.com/14272396/39008834-8afd1eb4-4412-11e8-887c-58ea18c89a51.png)
![image](https://user-images.githubusercontent.com/14272396/39008847-92c03faa-4412-11e8-8a51-e6cf2a4e9d92.png)
are unneeded here.
but i think it is bad, the we have logic in dialog, not in component